official/resnet/keras/imagenet_utils.py

```diff
@@ -0,0 +1,331 @@
+"""Utilities for ImageNet data preprocessing & prediction decoding.
```

Comment:
I might be missing something but IIUC the only use of this module is in keras/resnet_model.py where the only function being used is `_obtain_input_shape`, which in turn doesn't use any of the other functions defined in the file.

So can we nuke everything except `_obtain_input_shape` from this file? And if so, perhaps even nuke this file and inline `_obtain_input_shape` to the file in which it is called?

Reply:
Done.